src/Tree.cpp

```diff
@@ -8,19 +8,20 @@
 #endif
 
 Tree::
-Tree(const Rectangle r, int levels)
+Tree(const Rectangle& r, int levels)
 {
   rectangle_ = r;
-  cells_ = nullptr;
+  // cells_ = nullptr;
+  cells_.empty();
```

Comment:
This is not doing anything. It is just returning whether the vector is empty or not, but the output is not used.
Anyway the vector is necessarily empty here since is has not been filled.